migrations/helpers/OutputLogTemplate.js

```diff
@@ -0,0 +1,64 @@
+const logger = require('../../src/logger/');
+
+
+class OutputLogTempalte {
+	constructor({
+		total,
+		name,
+		_logger,
+		_detailInformations = false,
+		_lineBreak = '\n',
+	}) {
+		this.modified = [];
+		this.fail = [];
+		this.LF = _lineBreak;
+		this.total = total || 0;
+		this.l = _logger || logger;
```

Comment:
`logger` and others do not exist here...

Reply:
logger is import over require.